README.md

```diff
@@ -812,6 +820,81 @@ config.serialization_config.data_serializable_factories[1] = factory
 
 Note that the ID that is passed to the `SerializationConfig` is same as the factory ID that `Foo` class returns.
 
+### 4.2.3 - Versioning for Portable Serialization
```

Comment:
Change `### 4.2.3 - Versioning for Portable Serialization` to `### 4.2.1. Versioning for Portable Serialization` to follow the same standards through the README. Also, you should add this subtitle to table of contents